pkg/influx/builder.go

```diff
@@ -96,7 +93,6 @@ func (fb *FrameBuilder) Init(metadata *influxdb2.FluxTableMetadata) error {
 				return err
 			}
 			fb.value = converter
-		case col.Name() == "_measurement":
```

Comment:
timeseries check really depends on existance of `_value` field, not `_measurement`